### PR TITLE
#165273522: Set up Sass processing on the client

### PR DESCRIPTION
**Pull request by kalsmic**
#### What does this PR do?

- add SASS processing on the client
- add SASS/CSS linter

#### Description of Task to be completed?

- Configure Sass processing on the client and add Sass/CSS linter
#### How should this be manually tested?
- Visit the [review app](https://ah-frontend-dojo-pr-7.herokuapp.com/)

#### Any background context you want to provide?
We have set up sass processing on the client to be able to compile Sass to CSS
#### What are the relevant pivotal tracker stories?

[#165273522](https://www.pivotaltracker.com/story/show/165273522)

#### Screenshots (if appropriate)
Screenshot 2019-05-28 at 17 01 45

### CODE REVIEW

**Review by llwasampijja — APPROVED** [review]
@kalsmic nice work, I have checked the feature, and it is working fine. On that front, I think it can be merged!

**Review by bisonlou — APPROVED** [review]
Well done @kalsmic. I have checked out your work and I have seen the SASS take effect. This is a good foundation for our styling.

**Review by ZackAtama — APPROVED** [review]
Great work with setting up Sass @kalsmic

**Review by JamesMudidi — APPROVED** [review]
Thanks for the work @kalsmic, After testing it locally, I approve these changes

**Review by nadralia — APPROVED** [review]
Good work @kalsmic, you have set up a good foundation for styling

File: src/assets/MainStyle.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+* {
+  margin: 0;
```
[the remark this answers is not in the document]
Comment (kalsmic) [reply src/assets/MainStyle.scss:3]: This contradicts our CSS Style linting rules

**Pull request merged**